backend/src/main/java/com/woowacourse/levellog/team/presentation/TeamController.java

```diff
@@ -43,8 +44,16 @@ public ResponseEntity<TeamsDto> findAll() {
 
     @GetMapping("/{teamId}")
     @PublicAPI
-    public ResponseEntity<TeamDto> findById(@PathVariable final Long teamId) {
-        final TeamDto response = teamService.findById(teamId);
+    public ResponseEntity<TeamAndRoleDto> findById(@PathVariable final Long teamId, @Authentic final Long memberId) {
+        final TeamAndRoleDto response = teamService.findByTeamIdAndMemberId(teamId, memberId);
+        return ResponseEntity.ok(response);
+    }
+
+    @GetMapping("/{teamId}/members/{memberId}/my-role")
+    public ResponseEntity<InterviewRoleDto> findMyRole(@PathVariable final Long teamId,
+                                           @PathVariable(name = "memberId") final Long targetMemberId,
+                                           @Authentic final Long memberId) {
+        final InterviewRoleDto response = teamService.findMyRole(teamId, targetMemberId, memberId);
```

Comment:
명세랑 다르긴 하지만 `@PathVariable` 이름도 targetMemberId로 바꾸면 어떨까요?
`@Authentic`으로 반환하는 값이 memberId인데 PathVariable과 헷갈릴 우려가 있어서요.
```suggestion
    @GetMapping("/{teamId}/members/{targetMemberId}/my-role")
    public ResponseEntity<InterviewRoleDto> findMyRole(@PathVariable final Long teamId,
                                           @PathVariable final Long targetMemberId,
                                           @Authentic final Long memberId) {
        final InterviewRoleDto response = teamService.findMyRole(teamId, targetMemberId, memberId);
```